node/lib/util/checkout.js

```diff
@@ -116,49 +118,62 @@ exports.checkoutCommit = co.wrap(function *(metaRepo, commit) {
         }
     });
 
-    // Check meta
-
-    const metaError = yield dryRun(metaRepo, commit);
-    if (null !== metaError) {
-        errors.push(`Unable to check out meta-repo: ${metaError}.`);
-    }
-
-    // Try the submodules; store the opened repos and loaded commits for use
-    // in the actual checkout later.
-
     const cache = {};  // name to { repo, commit}
 
+    // Load repo/commit cache
```

Comment:
This method is getting long -- when you have named sections, it's often better to split them out into named methods.  Would that be possible here?